### PR TITLE
[DO-1755] Create PEP21 scoring rule

### PR DESCRIPTION
**Pull request by Andrii17s**
Rule 21
Monetary assets declared this year exceed the sum of income and amount of monetary assets of the previous year

### CODE REVIEW

**Review by roman-tiukh — COMMENTED** [review]
Do we have good logic for work with currencies?
I find 16 currencies for example:
```
    'UAH',
    'USD',
    'EUR',
    'RUB',
    'CHF',
    'GBP',
    'PLN',
    'CAD',
    'GEL',
    'CNY',
    'HRK',
    'SEK',
    'AUD',
    'TRY',
    'JPY',
    'HUF',
    'CZK',
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
blocked for now

File: business_register/pep_scoring/rules.py [hunk 1]
```diff
@@ -104,3 +104,84 @@ def calculate_weight(self):
...
+class IsMoneyFromNowhere(BaseScoringRule):
+    """
+    Rule 21 - PEP21
+    weight - 1.0
+    Monetary assets declared this year exceed the sum of
+    income and amount of monetary assets of the previous year
+    """
+
+    def calculate_weight(self):
+        declarations = Declaration.objects.filter(
```
Comment (IegorSoboliev) [inline business_register/pep_scoring/rules.py:118]: I think we have to use only the last two declarations for this rule

Comment (Andrii17s) [reply business_register/pep_scoring/rules.py]: @IegorSoboliev Why two, not just the last one?

**Pull request closed without merging**